x/subscription/keeper/cu_tracker.go

```diff
@@ -174,7 +175,14 @@ func (k Keeper) RewardAndResetCuTracker(ctx sdk.Context, cuTrackerTimerKeyBytes
 		// TODO: deal with the reward's remainder (uint division...)
 		providerAdjustment, ok := adjustmentFactorForProvider[provider]
 		if !ok {
-			providerAdjustment = sdk.OneDec().QuoInt64(int64(k.rewardsKeeper.MaxRewardBoost(ctx)))
+			maxRewardBoost := k.rewardsKeeper.MaxRewardBoost(ctx)
```

Comment:
what if we make sure this param is never zero, in validate params?

Reply:
I can add that too, but there are never too many sanity checks

Reply:
Also, how do I add a validation for params?

Reply:
thanks